sources/Application/Views/PhraseView.cpp

```diff
@@ -206,6 +207,12 @@ void PhraseView::updateCursorValue(ViewUpdateDirection direction,int xOffset,int
 	}
 	if ((c)&&(*c!=0xFF)) {
 		int offset=offsets_[col_+xOffset][direction] ;
+		// from: https://github.com/xiphonics/picoTracker/blob/master/sources/Application/Views/PhraseView.cpp
```

Comment:
I'd prefer if we keep this info in the corresponding .h and .cpp, it's not necessary to have it here.